README.md

```diff
@@ -4,95 +4,137 @@
 [![Container image](https://quay.io/repository/coreos/coreos-installer/status)](https://quay.io/repository/coreos/coreos-installer)
 [![crates.io](https://img.shields.io/crates/v/coreos-installer.svg)](https://crates.io/crates/coreos-installer)
 
-`coreos-installer` is a simple installer for Fedora CoreOS (FCOS) and Red Hat Enterprise Linux CoreOS (RHCOS).
-It contains minimal logic to download, verify and write an OS image to a target disk.
-It can be invoked as a standalone binary or during bootup via a dracut module.
+`coreos-installer` is a program to assist with installing Fedora CoreOS
+(FCOS) and Red Hat Enterprise Linux CoreOS (RHCOS). It can do the following:
 
-## Kernel command line options for coreos-installer running in the initramfs
+* Download and install the operating system to a target disk, optionally
+  customizing it with an Ignition config or first-boot kernel parameters
+  (`coreos-installer install`)
+* Download and verify an operating system image for various cloud,
+  virtualization, or bare metal platforms (`coreos-installer download`)
+* List Fedora CoreOS images available for download
+  (`coreos-installer list-stream`)
+* Embed an Ignition config in a live ISO image to customize the running
+  system that boots from it (`coreos-installer iso`)
 
-* `coreos.inst=yes` - Instruct the installer to run
-* `coreos.inst.install_dev` - The block device on the system to install to
-* `coreos.inst.image_url` - The URL of the CoreOS image to install to this device
-* `coreos.inst.ignition_url` - The URL of the CoreOS Ignition config (optional, enter
-  `coreos.inst.ignition_url=skip` to not load an Ignition config)
-* `coreos.inst.platform_id` - The Ignition platform ID of the platform the CoreOS image is being installed on
+The options available for each subcommand are available via the `--help`
+option.  This documentation will focus on how to obtain and run
+coreos-installer.
```

Comment:
This document mixes a bit of "coreos-installer" and "`coreos-installer`". Perhaps it would be better to pick one form and stick to it everywhere.